### DISCUSSION TITLE
curl: (28) SSL connection timeout

### DISCUSSION THREAD

**Discussion by Simplicity49** [Homebrew/discussions#5837] [Getting started]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

I can't install homebrew from terminal on my new macbook m1 max.

Error received from terminal: curl: (28) SSL connection timeout

Tried download pkg file from github in browser, it keeps timing out

**Comment by SMillerDev** [comment]
If you can't connect to GitHub there is nothing Homebrew can do to fix that.

**Reply by Simplicity49** [reply]
I can connect to github.

curl -v https://github.com/

    Connection #0 to host github.com left intact

**Reply by SMillerDev** [reply]
Yes, to that page. But clearly it's running out when you try and download the homebrew installer. There's nothing Homebrew can do about that.

**Reply by p-linnane** [reply]
The installer is at githubusercontent.com. Try to resolve that. Either way, it's an issue with your connection to GitHub, and we can't really do anything to help with that.
